deepmd/train/trainer.py

```diff
@@ -631,3 +655,33 @@ def _get_place_horders(self, data_dict):
                 prec = GLOBAL_ENER_FLOAT_PRECISION
             self.place_holders[kk] = tf.placeholder(prec, [None], name = 't_' + kk)
             self.place_holders['find_' + kk] = tf.placeholder(tf.float32, name = 't_find_' + kk)
+
+    def _init_from_frz_model(self):
+        # get the model type from the frozen model(self.run_opt.init_frz_model)
+        try:
+            t_model_type = get_tensor_by_name(self.run_opt.init_frz_model, 'model_type')
+            self.model_type = bytes.decode(t_model_type)
+        except GraphWithoutTensorError as e:
+            # throw runtime error if there's no frozen model
+            if os.path.exists(self.run_opt.init_frz_model) == False:
```

Comment:
Don't compare boolean values to True or False using ==. See [PEP-8](https://www.python.org/dev/peps/pep-0008/).
```suggestion
            if not os.path.exists(self.run_opt.init_frz_model):
```